### PR TITLE
3 packages from OCamlPro/ocplib-json-typed at 0.7

### PR DESCRIPTION
**Pull request by chambart**
This pull-request concerns:
-`ocplib-json-typed.0.7`: Type-aware JSON and JSON schema utilities
-`ocplib-json-typed-browser.0.7`: Json_repr interface over JavaScript's objects
-`ocplib-json-typed-bson.0.7`: A Json_repr compatible implementation of the JSON compatible subset of BSON

---
* Homepage: https://github.com/ocamlpro/ocplib-json-typed
* Source repo: git+https://github.com/ocamlpro/ocplib-json-typed.git
* Bug tracker: https://github.com/ocamlpro/ocplib-json-typed/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/ocplib-json-typed-browser/ocplib-json-typed-browser.0.7/opam [hunk 1]
```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+synopsis: "Json_repr interface over JavaScript's objects"
+maintainer: "Benjamin Canou <benjamin@ocamlpro.com>"
+authors: "Benjamin Canou <benjamin@ocamlpro.com>"
+homepage: "https://github.com/ocamlpro/ocplib-json-typed"
+bug-reports: "https://github.com/ocamlpro/ocplib-json-typed/issues"
+license: "LGPLv3 w/ linking exception"
+dev-repo: "git+https://github.com/ocamlpro/ocplib-json-typed.git"
+
+build: [ "dune" "build" "-j" jobs "-p" name "@install" ]
```
Comment (kit-ty-kate) [inline packages/ocplib-json-typed-browser/ocplib-json-typed-browser.0.7/opam]: the `@install` is unnecessary but for now it's fine as is

File: packages/ocplib-json-typed-bson/ocplib-json-typed-bson.0.7/opam [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+synopsis: "A Json_repr compatible implementation of the JSON compatible subset of BSON"
+maintainer: "Benjamin Canou <benjamin@ocamlpro.com>"
+authors: "Benjamin Canou <benjamin@ocamlpro.com>"
+homepage: "https://github.com/ocamlpro/ocplib-json-typed"
+bug-reports: "https://github.com/ocamlpro/ocplib-json-typed/issues"
+license: "LGPLv3 w/ linking exception"
+dev-repo: "git+https://github.com/ocamlpro/ocplib-json-typed.git"
+
+build: [ "dune" "build" "-j" jobs "-p" name "@install" ]
+run-test: [ "dune" "runtest" "-p" name "-j" jobs ]
```
Comment (avsm) [inline packages/ocplib-json-typed-bson/ocplib-json-typed-bson.0.7/opam]: this should be in the `build` stanza but annotated with `{with-test}` now

Comment (kit-ty-kate) [reply packages/ocplib-json-typed-bson/ocplib-json-typed-bson.0.7/opam]: mmh, both does the same thing as far as I am aware

Comment (avsm) [reply packages/ocplib-json-typed-bson/ocplib-json-typed-bson.0.7/opam]: then why do we have both... :-)

Comment (kit-ty-kate) [reply packages/ocplib-json-typed-bson/ocplib-json-typed-bson.0.7/opam]: I think the proper way is using `run-test` which runs the commands during the `test phase`. If you annotate with `{with-test}` it's done during the `build phase`. I think the annotation is primarily here to enable tests flags or something like that during configuration or compilation.

I'm not sure what's the difference between the `build phase` and the `test phase` (maybe there can be some differences from the bubblewrap point of view) but they are morally the same in the end, just one usage is more a distorted use from its original use I suppose.

cc @AltGr @rjbou

**Comment by chambart** [comment]
Indeed it doesn't

**Comment by kit-ty-kate** [comment]
Fair enough. Thanks

**Pull request merged**